File: main-api/src/main/java/italo/italomonitor/main/dto/integration/response/MonitorServerInfoResponse.java
package italo.italomonitor.main.dto.integration.response;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
@Builder
public class MonitorServerInfoResponse {

	private int numThreadsAtivas;
	
	private long totalMemory;
	
	private long freeMemory;
	
	private long maxMemory;
	
	private double usoCpu;
	
	private int availableProcessors;
	
}

File: frontend-app/src/core/model/dto/response/MonitorServerInfoResponse.ts

export interface MonitorServerInfoResponse {
    totalMemory : number;
	freeMemory : number;
	maxMemory : number;
	usoCpu : number;
	availableProcessors : number;
    numThreadsAtivas : number;
}

export const DEFAULT_MONITOR_SERVER_INFO_OBJ : MonitorServerInfoResponse = {
	totalMemory: 0,
	freeMemory: 0,
	maxMemory: 0,
	usoCpu: 0,
	availableProcessors: 0,
	numThreadsAtivas: 0
};